chatmm/src/main/java/com/likeminds/chatmm/chatroom/detail/model/LMChatAttachmentItemViewData.kt

```diff
@@ -0,0 +1,45 @@
+package com.likeminds.chatmm.chatroom.detail.model
+
+import android.os.Parcelable
+import androidx.annotation.DrawableRes
+import com.likeminds.chatmm.utils.model.BaseViewType
+import com.likeminds.chatmm.utils.model.ITEM_ATTACHMENT_ITEM
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+class LMChatAttachmentItemViewData private constructor(
```

Comment:
Please rename this to either `LMChatAttachmentBarViewData` or `LMChatAttachmentPickerViewData` or something else more meaningful, this name is confusing.

Reply:
Renaming as `LMChatAttachmentItemPickerViewData`